plugins/npm/test/npm-publish.test.ts

```diff
@@ -3,29 +3,24 @@ import NpmPublish from '../src/tasks/npm-publish'
 import winston, { Logger } from 'winston'
 import { ToolKitError } from '../../../lib/error/lib'
 import * as state from '@dotcom-tool-kit/state'
-import pacote, { ManifestResult } from 'pacote'
-import { publish } from 'libnpmpublish'
-import pack from 'libnpmpack'
+import { waitOnExit } from '@dotcom-tool-kit/logger'
+import { spawn } from 'node:child_process'
 
 const logger = (winston as unknown) as Logger
 
 const readStateMock = jest.spyOn(state, 'readState')
-jest.spyOn(pacote, 'manifest').mockImplementation(() => Promise.resolve({} as ManifestResult))
-jest.spyOn(process, 'cwd').mockImplementation(() => '')
-jest.mock('libnpmpack', () => {
-  return jest.fn(() => Promise.resolve())
-})
-jest.mock('libnpmpublish', () => {
-  return {
-    publish: jest.fn(() => Promise.resolve())
-  }
-})
+jest.mock('node:child_process', () => ({
+  spawn: jest.fn(() => Promise.resolve())
+}))
+jest.mock('@dotcom-tool-kit/logger', () => ({
+  waitOnExit: jest.fn(() => Promise.resolve())
+}))
```

Comment:
**suggestion**: this can be simplified to use jest's auto mocking

```suggestion
jest.mock('node:child_process')
jest.mock('@dotcom-tool-kit/logger')
```